CONTRIBUTING.md

```diff
@@ -0,0 +1,78 @@
+# How to contribute to proj-datumgrid
+
+Consult [PROJ CONTRIBUTING.md](https://github.com/OSGeo/proj.4/blob/master/CONTRIBUTING.md)
+for general principles.
+
+This document focuses on how to specifically contribute a new grid to proj-datumgrid.
+
+Preliminary:
+* The grid must be in one of the formats understood by PROJ, typically .gsb for horizontal shift
+grids and .gtx for vertical shift grids
+* The grid must be freely licensed as mentionned in [README.DATUMGRID](README.DATUMGRID).
+* We consider that you are familiar with the basic [GitHub workflow to submit a pull request](https://help.github.com/en/articles/creating-a-pull-request)
+
+Determine the geographic scope of the grid. We have one sub directory for
+continental packages
+- europe
+- north-america
+- oceania
+
+In the case of overseas territory, we have currently made the choice to attach grids
+to the continent of the mainland (hence all grids for French territories are in europe/,
+for USA in north-america/ etc.)
+
+Let's consider the case of adding a grid for a new European country.
+The steps are:
+1. Add it in the europe/ directory
```

Comment:
Formatting: `europe/`. Add some `'s around directory and file names. Improves readability.